nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2412,11 +2411,8 @@ private static ExpressionTree stripParensAndCasts(ExpressionTree expr) {
    * @return computed nullness for e, if any, else Nullable
    */
   public Nullness getComputedNullness(ExpressionTree e) {
-    if (computedNullnessMap.containsKey(e)) {
-      return computedNullnessMap.get(e);
-    } else {
-      return Nullness.NULLABLE;
-    }
+    Nullness nullness = computedNullnessMap.get(e);
```

Comment:
wanted to use `getOrDefault` here, but ran into https://github.com/uber/NullAway/issues/723

Reply:
Can we add a comment here to use `getOrDefault` once it is supported properly?

Reply:
added but this pattern exists in a few other places as well, i had to undo my changes to them (see first force-push on this PR)